agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -242,6 +242,55 @@ if [ $lcl_cnt -gt 0 ]; then
 fi
 wait
 
+stockpile_path=$(getconf.py -C $configfile stockpile_path stockpile)
+stockpile_user=$(getconf.py -C $configfile stockpile_user stockpile)
+stockpile_local_remote_user=$(getconf.py -C $configfile local_remote_user stockpile)
+stockpile_host_remote_user=$(getconf.py -C $configfile host_remote_user stockpile)
+stockpile_output_path=$(getconf.py -C $configfile stockpile_output_path stockpile)
+
+# Install stockpile and its dependencies
+which git &>/dev/null
```

Comment:
As discussed with @portante and @ndokos, the dependencies should be just installed on the controller, not on all the remotes. We are working on a common approach to handle the dependencies during the runtime.